src/graphql/queries/ListArticles.js

```diff
@@ -85,7 +90,7 @@ export default {
     };
 
     // Collecting queries that will be used in bool queries later
-    const mustQueries = []; // Affects scores
+    const shouldQueries = []; // Affects scores
```

Comment:
What does `shouldQueries` do?

Reply:
It's the `should` clause of ElasticSearch bool query
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html